### PR TITLE
Component library Select component

### PR DESCRIPTION
**Pull request by bradfordjanice**
The CIVIC Select component.  Ready for review.  
The component is complete, except for adding the styling using the new theme.
Stories have been written for Storybook.

### CODE REVIEW

**Comment by bradfordjanice** [comment]
Don't understand the conflict and how to correct it.

**Comment by jaronheard** [comment]
Here's the magic sauce to fix the merge conflict:
(starting on your branch, in the root directory of the repo)
`git merge origin/master`
merge will fail, you'll have 300+ modified files, and git status will say you are 300+ commits ahead of master. remain calm. go to the file with the merge conflict, fix it, and save.
`git add .`
`git commit --no verify`
you'll get a merge message to modify, just keep that message the same and save and close
`git push origin component-library/Dropdown`

File: packages/component-library/stories/Select.story.js [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                    formLabel={formLabel}
+                    onChange={event => {
+                      set({ value: event.target.value });
+                      action("onChange")(event);
+                    }}
+                    value={get("value")}
+                    disabled={disabled}
+                    options={options}
+                  />
+                  <FormHelperText>{formHelperText}</FormHelperText>
```
Comment (jaronheard) [inline packages/component-library/stories/Select.story.js]: FormHelperText should be implemented in Select.js by adding an optional `helperText` prop -- One of our goals is that the developer using our component shouldn't have to know anything about Material to use our component.

**Comment by jaronheard** [comment]
@bradfordjanice I see some new commits, is this now ready for a re-review?

**Comment by bradfordjanice** [comment]
Yes, ready for review.  I will make a different branch/pull request when I add the styling.

**Pull request merged**